cliquet/tests/support.py

```diff
@@ -86,25 +86,29 @@ class BaseWebTest(FakeAuthentMixin):
 
     api_prefix = "v0"
 
-    def get_test_app(self):
-        return webtest.TestApp(testapp(self.get_app_settings()))
+    def get_test_app(self, settings=None):
+        app = webtest.TestApp(testapp(self.get_app_settings(settings)))
+        app.RequestClass = get_request_class(self.api_prefix)
+        return app
```

Comment:
The fact that this method has "test" in its name makes nose executing it as a test :(

Elsewhere we use patch.dict(), we should make it consistent to use the same method everywhere

I noticed that instantiating app was slow, and took it out in #192. This point makes this PR conflicting with #192 then